js/ui/map.js

```diff
@@ -1311,6 +1354,9 @@ function removeNode(node) {
  * @memberof Map
  * @instance
  * @property {MapMouseEvent} data
+ * @see [Measure distances](https://www.mapbox.com/mapbox-gl-js/example/measure/)
+ * @see [Navigate the map with game-like controls](https://www.mapbox.com/mapbox-gl-js/example/game-controls/)
```

Comment:
This example didn't need to use the `click` event. I simplified the example in https://github.com/mapbox/mapbox-gl-js/commit/6ae57801e452185b5472846b977fc56204b0780b